pkg/kubelet/kubelet.go

```diff
@@ -151,9 +151,12 @@ type Kubelet struct {
 	podInfraContainerImage string
 	podWorkers             *podWorkers
 	resyncInterval         time.Duration
-	pods                   []api.BoundPod
 	sourceReady            SourceReadyFn
 
+	// Protectes the pods array
```

Comment:
nit: s/tes/ts/

Also consider adding another line ```// We make shallow copies out of this while locked, which is OK because once added to this array, pods are immutable``` or something to that effect.